plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -209,11 +186,105 @@ class NetworkingFormSection extends mixin(StoreMixin) {
                 </Tooltip>
               </FormGroupHeadingContent>
             </FormGroupHeading>
-            <FieldHelp>
-              Load balance this service internally at {hostName}
-            </FieldHelp>
           </FieldLabel>
-          <FieldError>{vipPortError || loadBalancedError}</FieldError>
+        </FormGroup>
+      </FormRow>,
+      (loadBalanced && this.getLoadBalancedPortField(endpoint, index))
+    ];
+  }
+
+  getLoadBalancedPortField(endpoint, index) {
```

Comment:
I tried to keep the style in this 1.9 fix though the first intention was to extract components. That would have resulted in too many changes that don't have anything to do with the bug. So I decided not to sneak a refactoring along with this bug fix.